scripts/wallet-sanity.sh

```diff
@@ -33,6 +33,25 @@ pay_and_confirm() {
   $solana_wallet "${entrypoint[@]}" confirm "$signature"
 }
 
+leader_readiness=false
```

Comment:
Did you consider doing something simpler like trying to airdrop N times until it works?  I feel that might accomplish the same goal of determining if the network is running without building in another command

Reply:
Yes, I did consider that, and it would certainly work. My thinking was that the airdrop was part of the test mechanism of the script and so should be distinct. There's also a lot more going on with the airdrop (drone, leader, tpu, polling balance), so if it fails, it is harder to diagnose. I thought it might be valuable to separate out this one failure case.

Let me know what you think, @mvines . I can certainly rework that direction